abdulfataiaka/pluralsight-admin
# #161284664 Write tests for implementations

#### What does this PR do
- Writes tests for implementations

#### Description of work done
- Write component tests

#### How should it be manually tested
- Clone the repository `git clone https://github.com/abdulfataiaka/pluralsight-admin.git`
- Checkout to the branch `ft-test-implementations-161284664`
-  Run `npm test`
- Ensure all tests are passing

#### Relevant pivotal tracker story
[161284664](https://www.pivotaltracker.com/story/show/161284664)
